maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/util/IOUtils.java

```diff
@@ -187,4 +188,14 @@ private static String normalizePattern(String pattern)  {
 
 		return pattern;
 	}
+
+    public static void close(URLClassLoader classLoader) {
```

Comment:
Not needed. Use try-with-resources pattern instead

Reply:
Deleted code